### PR TITLE
Switch

### PR DESCRIPTION
**Pull request by ChaitanyaGadodia**
design: https://app.zeplin.io/project/5ca592f2b94a0d6c24fa6f95/screen/5ca59318d5d005a22478366d

### CODE REVIEW

File: packages/pebble-web/src/components/Toggle.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  switchStyle
+} from "./styles/Toggle.styles";
+
+interface ToggleProps {
+  initialValue?: boolean;
+  onChange?: (value: boolean) => void;
+}
+
+interface ToggleState {
+  value?: boolean;
```
Comment (azizhk) [inline packages/pebble-web/src/components/Toggle.tsx]: Why is this not a controlled component?
All of pebble's other components are controlled.

**Comment by azizhk** [comment]
Also please look into ensuring that it is accessible and has big touch areas for mobile also

**Comment by ChaitanyaGadodia** [comment]
> Also please look into ensuring that it is accessible and has big touch areas for mobile also

@jekingala

**Comment by azizhk** [comment]
@ChaitanyaGadodia This needs to be ensured by frontend.

**Comment by ChaitanyaGadodia** [comment]
> @ChaitanyaGadodia This needs to be ensured by frontend.

Yes, the component looks small, so need designs input.

**Comment by ritz078** [comment]
Switches are generally for Toggle. So you should consider renaming this to just `Switch`.

**Pull request merged**